CONTRIBUTING.md

```diff
@@ -33,7 +33,15 @@ Follow these steps to build a new version of the plugin:
 
 ### Environment setup
 
-Make sure you can build the plugin by following the steps above.
+First, make sure you can build the plugin by following the steps above.
+
+Then, make sure you have installed ```Gulp```, ```Mocha```, and ```Mocha JUnit Reporter``` through npm.
+
+```
+npm install -g gulp
+npm install -g mocha
+npm install -g mocha-junit-reporter
```

Comment:
hmm gulp is already in devdependencies but mocha is not...I recall having to install gulp and mocha globally to get it to work in the first place, i'll investigate how changing the devdependencies affects it